buildSrc/subprojects/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt

```diff
@@ -153,6 +153,7 @@ abstract class ExternalModulesExtension {
     val sshdSftp = "org.apache.sshd:sshd-sftp"
     val testcontainersSpock = "org.testcontainers:spock"
     val xmlunit = "xmlunit:xmlunit"
+    val joptSimple = "net.sf.jopt-simple:jopt-simple"
```

Comment:
This list is supposed to be sorted alphabetically.

Reply:
Done.